Library/Homebrew/cmd/test-bot.rb

```diff
@@ -573,11 +573,16 @@ def formula(formula_name)
     def homebrew
       @category = __method__
       return if @skip_homebrew
-      test "brew", "tests"
-      test "brew", "tests", "--no-compat" if @tap
-      readall_args = ["--aliases"]
-      readall_args << "--syntax" if MacOS.version >= :mavericks
-      test "brew", "readall", *readall_args
+      if @tap
+        test "brew", "readall", @tap
```

Comment:
Last time we talked about this I said I thought it made sense to run `brew tests` for taps so they can see if there are upstream issues. Do you disagree?